packages/front-end/components/Experiment/ResultMoreMenu.tsx

```diff
@@ -114,47 +122,42 @@ export default function ResultMoreMenu({
           Report
         </Button>
       )}
+      <Tooltip
+        shouldDisplay={!canDownloadJupyterNotebook}
+        body="To download results as a Jupyter notebook, you must set up a Jupyter Notebook query runner. View our docs for more info."
```

Comment:
I would love for that to be possible, but the way the `Tooltip` component is currently built, if you move your mouse to click on the link, the Tooltip disappears. I'll create an issue for us to revisit the Tooltip's logic as there are quite a few areas in the app where this would come in handy.